---
VLP16_SettingsMessage  msg
